index.html

```diff
@@ -28,7 +28,7 @@
       <h1 class="hero-heading">Cat Match</h1>
       <p class="hero-sub-heading">Find your perfect cat breed</p>
       <div class="hero-svg-section">
-        <img src="./assets/images/man_cat_couch.svg" class="hero-svg" alt="man and cat on couch">
+        <img src="./assets/images/man_cat_couch.svg" class="hero-svg" alt="person and cat sitting on a couch">
```

Comment:
hey @Jaytru1997, image is not loading as we're still trying to access it from its 'old' name and not the one you've renamed it to